crates/subspace-farmer/src/dsn/tests.rs

```diff
@@ -197,7 +200,7 @@ async fn test_dsn_sync() {
                     last_archived_block,
                 };
 
-                let mut pieces = FlatPieces::new(256);
+                let mut pieces = FlatPieces::new(pieces_per_segment as _);
```

Comment:
```suggestion
                 let mut pieces = FlatPieces::new(pieces_per_segment as usize); 
```

Reply:
Do we actually care about type of numbers in tests? Type inference seem to handle it well

Reply:
It is just a few characters and removes a question of the type during review completely. I understand when it is used for complex types or types that can't be named or need to be imported just to specify them, but here I see no reason to not type it explicitly.

Not a blocker of course, just a matter of taste.